docs/getting-started.md

```diff
@@ -8,8 +8,11 @@ You can check if your system meets the requirements by running
 An opinionated way to run Firecracker is to launch an
 [EC2](https://aws.amazon.com/ec2/) `c5.metal` instance with Ubuntu 22.04.
 
-EC2 only supports nested virtualization on metal instances, which is why we use
-`.metal` instances exclusively.
+Firecracker requires [the KVM Linux kernel module](https://www.linux-kvm.org/)
+to perform some of its virtualization and emulation tasks.
```

Comment:
The wording here may raise the question

> What virtualization and emulation tasks can Firecracker do without KVM?

My understanding is our answer is none.

I would suggest:

```suggestion
Firecracker requires [the KVM Linux kernel module](https://www.linux-kvm.org/).
```

Reply:
Please refer to this comment https://github.com/firecracker-microvm/firecracker/pull/4423#discussion_r1477815861 for latest updates.

We all agree that Firecracker need KVM for dong anything meaningful that it was the spiritn of the comment. I still want to keep the sentence because it gives color and explain the "why" that is important for any novice.

Reply:
The new wording covers my concerns.